airgun/views/rhai.py

```diff
@@ -10,3 +12,28 @@ class AllRulesView(BaseLoggedInView):
     @property
     def is_displayed(self):
         return self.title.is_displayed and self.search.is_displayed
+
+
+class InventoryAllHosts(BaseLoggedInView):
+    search = TextInput(locator=".//input[@placeholder='Find a system']")
+    actions = Dropdown("Actions")
```

Comment:
is that `actions` part of a table? if yes - then it should be a part of column_widget
But anyway, we use `ActionsDropdown` for both situations